Code/autopkglib/github/__init__.py

```diff
@@ -274,6 +274,9 @@ def call_api(
         try:
             with open(temp_content) as f:
                 resp_data = json.load(f)
+        except UnicodeDecodeError:
+            with open(temp_content, "rb") as f:
+                resp_data = json.load(f)
         except json.JSONDecodeError:
             resp_data = None
```

Comment:
maybe `resp_data` should be set to `None` before this try/except block on Line 273? Then this last except could just be a `pass`?

Reply:
Good catch, that's ugly code. That definitely doesn't work for all the cases.